apex-app-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml

```diff
@@ -50,6 +50,12 @@
       </includes>
     </fileSet>
     <fileSet filtered="true" encoding="UTF-8">
+      <directory>src/main/resources</directory>
+      <includes>
+        <include>**/log4j.props</include>
```

Comment:
When I name properties file as log4j.properties, it ends up picking hadoop log4j.properties. I tried to change classpath by adding folder having our log4j.properties at end/start but was facing some issues. So to avoid any of such problems I changed name to log4j.props
In fact now I am thinking I should change it even further to name it like apexlog4j.properties/props